pkg/kratos/service.go

```diff
@@ -417,8 +426,11 @@ func (s *Service) getUiError(responseBody io.ReadCloser) (err error) {
 		err = fmt.Errorf("invalid backup code")
 	case MissingBackupCodesSetup:
 		err = fmt.Errorf("login with backup codes unavailable")
+	case PasswordIdentifierSimilarity:
+		err = fmt.Errorf("password can not be similar to the email")
 	default:
-		err = fmt.Errorf("unknown kratos error code: %v", errorCode)
+		s.logger.Debugf("Unknown kratos error code: %v", errorCode)
```

Comment:
personal preference, feel free to disregard: I don't think `debug` is the correct log level for this, IMO I would want to see the error code even in a production env